sqllogictest/src/runner.rs

```diff
@@ -134,7 +134,7 @@ pub enum TestErrorKind {
         sql: String,
         err: Arc<dyn std::error::Error + Send + Sync>,
     },
-    #[error("query result mismatch:\n[SQL] {sql}\n[Diff]\n{}", difference::Changeset::new(.expected, .actual, "\n"))]
+    #[error("query result mismatch:\n[SQL] {sql}\n[Diff]\n{}", difference::Changeset::new(.actual, .expected, "\n"))]
```

Comment:
What about
```suggestion
    #[error(
        "query result mismatch:\n[SQL] {sql}\n[Diff] (\x1b[91m-excepted\x1b[0m|\x1b[92m+actual\x1b[0m)\n{}",
        difference::Changeset::new(.expected, .actual, "\n").diffs.iter().format_with("\n", |diff, f| {
            match *diff {
                Difference::Same(ref x) => {
                    f(&format_args!(" {}", x))
                }
                Difference::Add(ref x) => {
                    f(&format_args!("\x1b[92m+{}\x1b[0m", x))
                }
                Difference::Rem(ref x) => {
                    f(&format_args!("\x1b[91m-{}\x1b[0m", x))

                }
            }
        })
    )]
```

<img width="377" alt="image" src="https://user-images.githubusercontent.com/37948597/203513296-dd78bd31-9650-468f-8307-4a8adbc235f8.png">

Reply:
Look cool!